src/backend/distributed/sql/udfs/citus_rebalance_start/latest.sql

```diff
@@ -0,0 +1,10 @@
+CREATE OR REPLACE FUNCTION pg_catalog.citus_rebalance_start(
+        rebalance_strategy name default NULL,
+        shard_transfer_mode citus.shard_transfer_mode default 'auto'
+    )
+    RETURNS VOID
+    AS 'MODULE_PATHNAME'
+    LANGUAGE C VOLATILE;
+COMMENT ON FUNCTION pg_catalog.citus_rebalance_start(name, citus.shard_transfer_mode)
+    IS 'rebalance the shards in the cluster in the background';
+GRANT EXECUTE ON FUNCTION pg_catalog.citus_rebalance_start(name, citus.shard_transfer_mode) TO PUBLIC;
```

Comment:
at which point do we check permissions? (I guess we should avoid scheduling tasks)

Reply:
Need to double check where permissions are checked in the old rebalancer. That might actually only happen on the move.

Tasks schedule as the current user, so it would error at some point, but that better be before we schedule the tasks indeed.

Reply:
added ownership check for tables in moving colocation groups.